.github/workflows/deploy.yml

```diff
@@ -21,7 +21,6 @@ jobs:
 
   conditions:
     name: Check for cluster conditions
-    needs: build-forms-shared
```

Comment:
now forms shared are not build on every run, but only on those instances where are needed